namespace name 'Enums' does not exist

Hello,
Error(s) : reference crosscuting enums
     SampleEntity.cs(5,30): error CS0234: The type or namespace name 'Enums' does not exist in the namespace 'MyCompany.Crosscutting' (are you missing an assembly reference?) [/dotnetcoresample/src/Dotnetcoresample.Domain/Dotnetcoresample.Domain.csproj]

##### **JHipster Version(s)**

```
/Users/developer
└── generator-jhipster@6.10.1 

```


##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**


<details>
<summary>.yo-rc.json file</summary>
<pre>
{
  "generator-jhipster": {
    "promptValues": {
      "nativeLanguage": "en"
    },
    "namespace": "MyCompany",
    "databaseType": "sqllite",
    "authenticationType": "jwt",
    "prodDatabaseType": "mysql",
    "jhipsterVersion": "6.10.0",
    "applicationType": "monolith",
    "baseName": "dotnetcoresample",
    "useSass": true,
    "clientPackageManager": "npm",
    "clientFramework": "angularX",
    "clientTheme": "none",
    "clientThemeVariant": "",
    "creationTimestamp": 1596093870148,
    "testFrameworks": [],
    "jhiPrefix": "jhi",
    "entitySuffix": "",
    "dtoSuffix": "DTO",
    "otherModules": [
      {
        "name": "generator-jhipster-dotnetcore",
        "version": "1.0.0"
      }
    ],
    "enableTranslation": true,
    "nativeLanguage": "en",
    "languages": ["en", "fa"],
    "blueprints": [
      {
        "name": "generator-jhipster-dotnetcore",
        "version": "1.0.0"
      }
    ]
  }
}

</pre>
</details>


##### **JDL for the Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

<details>
<summary>JDL entity definitions</summary>

<pre>

entity SampleEntity {
  names String,
  dbdate LocalDate required
}
relationship ManyToOne {
  NewSample{user(login)} to User,
}

dto NewSample with mapstruct
service  SampleEntity with serviceImpl

</pre>
</details>


##### **Environment and Tools**

openjdk version "1.8.0_242"
OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_242-b08)
OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.242-b08, mixed mode)

git version 2.20.1

node: v13.7.0

npm: 6.14.4

yeoman: 3.1.1

yarn: 1.21.1

Docker version 19.03.8, build afacb8b

docker-compose version 1.25.4, build 8d51620a

identical .jhipster/SampleEntity.json

--- a/generators/entity-server/templates/dotnetcore/src/Project/Controllers/EntityController.cs.ejs
+++ b/generators/entity-server/templates/dotnetcore/src/Project/Controllers/EntityController.cs.ejs
@@ -29,7 +29,16 @@ using JHipsterNet.Pagination.Extensions;
 using <%= namespace %>.Data;
 using <%= namespace %>.Data.Extensions;
 using <%= namespace %>.Domain;
+<%_
+let hasEnumField = false;
+fields.forEach(field => {
+        if (field.fieldIsEnum) {
+            hasEnumField = true;
+        }
+    });
+if (hasEnumField) { _%>
 using <%= namespace %>.Crosscutting.Enums;
+<%_ } _%>
 <%_ if (hasDto) { _%>
 using <%= namespace %>.Dto;
 using <%= namespace %>.Service.Mapper;

--- a/generators/entity-server/templates/dotnetcore/src/Project/Data/ApplicationDatabaseContext.cs.ejs
+++ b/generators/entity-server/templates/dotnetcore/src/Project/Data/ApplicationDatabaseContext.cs.ejs
@@ -13,7 +13,18 @@
  limitations under the License.
 -%>
 using <%= namespace %>.Domain;
+<%_
+let hasEnumField = false;
+entities.forEach(entity => {
+    fields.forEach(field => {
+        if (field.fieldIsEnum) {
+            hasEnumField = true;
+        }
+    });
+});
+if (hasEnumField) { _%>
 using <%= namespace %>.Crosscutting.Enums;
+<%_ } _%>
 using Microsoft.AspNetCore.Identity;
 using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore;

--- a/generators/entity-server/templates/dotnetcore/src/Project/Models/Entity.cs.ejs
+++ b/generators/entity-server/templates/dotnetcore/src/Project/Models/Entity.cs.ejs
@@ -25,7 +25,16 @@ using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
+<%_
+let hasEnumField = false;
+fields.forEach(field => {
+        if (field.fieldIsEnum) {
+            hasEnumField = true;
+        }
+    });
+if (hasEnumField) { _%>
 using <%= namespace %>.Crosscutting.Enums;
+<%_ } _%>
 <%_ if (hasManyToMany) { _%>
 using <%= namespace %>.Domain.RelationshipTools;
 <%_ } if (importJsonIgnore === true || hasManyToMany) { _%>

--- a/generators/entity-server/templates/dotnetcore/test/Project.Test/Controllers/EntityResourceIntTest.cs.ejs
+++ b/generators/entity-server/templates/dotnetcore/test/Project.Test/Controllers/EntityResourceIntTest.cs.ejs
@@ -131,7 +131,7 @@ using System.Threading.Tasks;
 using FluentAssertions;
 using <%= namespace %>.Data;
 using <%= namespace %>.Domain;
-<<%_
+<%_
 let hasEnumField = false;
 fields.forEach(field => {
         if (field.fieldIsEnum) {

--- a/generators/entity-server/templates/dotnetcore/test/Project.Test/Controllers/EntityResourceIntTest.cs.ejs
+++ b/generators/entity-server/templates/dotnetcore/test/Project.Test/Controllers/EntityResourceIntTest.cs.ejs
@@ -131,7 +131,16 @@ using System.Threading.Tasks;
 using FluentAssertions;
 using <%= namespace %>.Data;
 using <%= namespace %>.Domain;
+<<%_
+let hasEnumField = false;
+fields.forEach(field => {
+        if (field.fieldIsEnum) {
+            hasEnumField = true;
+        }
+    });
+if (hasEnumField) { _%>
 using <%= namespace %>.Crosscutting.Enums;
+<%_ } _%>
 <%_ if (hasDto) { _%>
 using <%= namespace %>.Dto;
 using <%= namespace %>.Service.Mapper;

